src/bin/gpfdist/Makefile

```diff
@@ -60,9 +60,11 @@ mkgpfdist: mkdir
 
 	for file in $(GPFDISTFILES); do \
 	    ( $(CC) $(INCLUDES) $(CFLAGS) $(LIBS) -c $(code_dir)$${file}); \
```

Comment:
Get it. But I have no test env for retest this write, I will keep it unchanged because the previous writing was already verified.